ScoutSuite/providers/gcp/facade/utils.py

```diff
@@ -0,0 +1,19 @@
+import httplib2
+import googleapiclient
+from ScoutSuite.providers.utils import run_concurrently
+
+class GCPFacadeUtils:
+    @staticmethod
+    def _get_all(resources, resource_key, request, resources_group):
```

Comment:
Are there  parameters you can add type hints to?

Reply:
I didn't find any types for `resources`, `request` and `resources_group` as the library is auto-generated using the Google Cloud API description.